package.json

```diff
@@ -18,22 +18,29 @@
         "quick-lru": "^5.1.1"
     },
     "devDependencies": {
-        "@apify/eslint-config": "^0.1.3",
+        "@apify/eslint-config-ts": "^0.1.2",
+        "@types/node": "^16.4.13",
+        "@typescript-eslint/eslint-plugin": "^4.29.0",
+        "@typescript-eslint/parser": "^4.29.0",
         "body-parser": "^1.19.0",
+        "del-cli": "^4.0.1",
         "eslint": "^7.0.0",
         "express": "^4.17.1",
         "fs-extra": "^9.1.0",
         "get-stream": "^5.2.0",
         "jest": "^26.6.3",
         "jest-extended": "^0.11.5",
         "jsdoc-to-markdown": "^7.0.0",
-        "markdown-toc": "^1.2.0"
+        "markdown-toc": "^1.2.0",
+        "typescript": "^4.3.5"
     },
     "scripts": {
+        "build": "del-cli dist && tsc",
+        "prepare": "npm run build",
```

Comment:
Why is this a script? Isn't it redundant, especially for dev envs?

Reply:
tl;dr runs after `npm install` and before `npm publish`

Reply:
That's a good one. In the SDK we've been using the `prepare` script for as long as I remember. Tbh I kinda hate it because it slows installs down, but at the same time, you don't end up running against some old build accidentally.

I think the best solution would be always to run tests against source without the need to build and then we most likely would not need this `prepare` script and we could use `prepublishOnly` and save some seconds on install. So yeah, agree with Vlad, but we first need to convert all tests to TS.

Reply:
> I kinda hate it because it slows installs down, but at the same time

How does it slow down? You have to build the project anyway.

> I think the best solution would be always to run tests against source without the need to build

Do you mean to use `ts-node`?

Reply:
Look in browser-pool for tests stuffs, you don't have to use ts-node (directly at least)
You'll just need to update the jest config, install the deps, and import from src directly 👍